src/Tests/Execute/Device.test.ts

```diff
@@ -20,8 +20,8 @@ import {DeviceSpec, HostPCSpec, TizenDeviceSpec} from '../../Backend/Spec';
 import {Device} from '../../Execute/Device';
 
 suite('Device', function() {
-  suite('Device', function() {
-    suite('#contructor()', function() {
+  suite('#Device', function() {
+    test('contructor', function() {
```

Comment:
Sorry for bothering but could you use unique test name even they are different suite?
Our tool has a bug :(

Reply:
Oh I will fix it.